Laba №5 Huffman Coding/HuffmanTree.h

```diff
@@ -0,0 +1,71 @@
+#pragma once
+
+#include "/O-O-P/boolVector/boolVector.h"
+#include <string>
+#include <vector>
+#include <algorithm>
+#include <fstream>
+
+
+class HuffmanTree
+{
+	class Node
+	{
+	public:
+		Node();
+		~Node() = default;
+
+		std::string stringifySymbols() const;
+
+		int frequency() const;
+		void setFrequency(int frequency);
+
+		BoolVector symbols() const;
+		void setSymbols(BoolVector symbols);
+
+		Node* leftChild() const;
+		Node* rightChild() const;
+
+		void setLeftChild(Node* leftChild);
+		void setRightChild(Node* rightChild);
+
+		void printHorizontal(Node* root, int marginLeft, int levelSpacing) const;
+
+		BoolVector m_symbols = BoolVector(256, 0);
+
+	private:
+
+
+		int m_frequency = 1;
+
+		Node* m_leftChild = nullptr;
+		Node* m_rightChild = nullptr;
+
+	};
+public:
+	HuffmanTree() = default;
+	~HuffmanTree();
+
+	void clear();
+
+	void print();
+
+	void build(const std::string& text);

```

Comment:
Я так понимаю, в срр название исправили, а тут забыли.
```suggestion
	void build(const std::string& textFilename);
```